client/verta/docs/api/api/lock.rst

```diff
@@ -0,0 +1,19 @@
+.. lock-api:
+
+Lock
+====
+
+.. note::
+    .. versionadded:: 0.17.3
+
+.. autoclass:: verta.lock.Closed
```

Comment:
`lock` as a top level package is very weird to me. I was expecting `verta.registry.lock` or something like that. Sorry if I missed this in the original proposal

Reply:
Oops that's a typo! It is supposed to be `verta.registry.lock`
Thanks!